docs/jwst/assign_wcs/reference_files.rst

```diff
@@ -29,53 +29,132 @@ reftype                                     description
 
-:model: Transform through the COLLIMATOR.
+FILTEROFFSET
+::::::::::::
+For MIRI, FILTEROFFSET reference files are currently selected based on the values of EXP_TYPE and DETECTOR in the input science data set.
 
-DISPERSER
----------
+FORE
+::::
 
-CRDS Selection Criteria
-:::::::::::::::::::::::
+For NIRSPEC, FORE reference files are currently selected based on the values of EXP_TYPE and FILTER in the input science data set.
+
+FPA
+:::
+For NIRSPEC, FPA reference files are currently selected based only on the value of EXP_TYPE in the input science data set.
+
+IFUFORE
+:::::::
+For NIRSPEC, IFUFORE reference files are currently selected based only on the value of EXP_TYPE in the input science data set.
+
+
+IFUPOST
+:::::::
+For NIRSPEC, IFUPOST reference files are currently selected based only on the value of EXP_TYPE in the input science data set.
+
+IFUSLICER
+:::::::::
+For NIRSPEC, IFUSLICER reference files are currently selected based only on the value of EXP_TYPE in the input science data set.
+
+
+MSA
+:::
+For NIRSPEC, MSA reference files are currently selected based only on the value of EXP_TYPE in the input science data set.
+
+OTE
+:::
+For NIRSPEC, OTE reference files are currently selected based only on the value of EXP_TYPE in the input science data set.
+
+SPECWCS
+:::::::
+For MIRI, SPECWCS reference files are currently selected based on the values of DETECTOR, CHANNEL, BAND, SUBARRAY, and EXP_TYPE in the input science data set.
+
+For NIRCAM, SPECWCS reference files are currently selected based on teh values of EXPT_TYPE, MODULE, and PUPIL in the input science data set.
+
+For NIRISS GRISM and TSGRIM modes, SPECWCS reference files are currently selected based on the values of SUBARRAY and EXP_TYPE in the input science data set.
+
+FOR NIRISS WFSS mode, SPECWCS reference files are currently selected based on the values of EXP_TYPE, FILTER, and PUPIL in the input science data set.
+
+
+
+
+REGIONS
+:::::::
+For MIRI, REGIONS reference files are currently selected based on the values of DETECTOR, CHANNEL, BAND, EXP_TYPE in the input science data set.
+
+
+V2V3
+::::
+For MIRI, V2V3 reference files are currently selected based on the values of DETECTOR, CHANNEL, BAND, EXP_TYPE in the input science data set.
+
+
+WAVELENGTHRANGE
+:::::::::::::::
+For NIRSPEC, WAVELENGTHRANGE reference files are currently selected based only on the value of EXP_TYPE in the input science data set.
+
+For MIRI, WAVELENGTHRANGE reference files are currently selected based only on the value of EXP_TYPE in the input science data set.
+
+For NIRCAM, WAVELENGTHRANGE reference files are currently selected based only on the value of EXP_TYPE in the input science data set.
+
+For NIRISS, WAVELENGTHRANGE reference files are currently selected based only on the value of EXP_TYPE in the input science data set.
```

Comment:
Could this whole block be consolidated to say:
```
For MIRI, NIRCam, NIRISS, and NIRSpec WAVELENGTHRANGE reference files are ...
```
because they're all the same. If we ever need to change any one of them, the list could be rexpanded then.